mythtv/programs/mythbackend/encoderlink.cpp

```diff
@@ -330,8 +330,10 @@ bool EncoderLink::CheckFile(ProgramInfo *pginfo)
         ReferenceLocker rlocker(m_sock);
         return m_sock->CheckFile(pginfo);
     }
+    #ifndef _WIN32 // Needs fixing for Windows
+        pginfo->SetPathname(GetPlaybackURL(pginfo));
+    #endif
 
-    pginfo->SetPathname(GetPlaybackURL(pginfo));
```

Comment:
Is the problem with pginfo or SetPathname? There are a bunch of calls to ProgramInfo::SetPathname from various files. Would it be better to put the #ifdef there instead of making this change (and other similar changes), and then having to revert them later?

Reply:
Added TODO into the code where appropriate. I have removed the if not windows condition in this specific example. 
There is a undefined reference error to GetPlayBackURL function and for compiling purpose's a dummy GetPlaybackURL was added.